frontend/apps/marketing/src/components/richText/RichText.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  documentToReactComponents,
+  Options,
+} from '@contentful/rich-text-react-renderer';
+import {
+  BLOCKS,
+  INLINES,
+  MARKS,
+  Document,
+  Block,
+  Inline,
+  Text,
+  Mark,
+} from '@contentful/rich-text-types';
+import {EntryFields} from 'contentful';
+import {ReactNode} from 'react';
+
+import Link from '@code-dot-org/component-library/link';
+import SimpleList, {
+  SimpleListItem,
+} from '@code-dot-org/component-library/list/simpleList';
+import {BodyTwoText} from '@code-dot-org/component-library/typography';
+
+import moduleStyles from './richText.module.scss';
+
+export interface RichTextProps {
+  content?: EntryFields.RichText | Document;
+}
+
+export type RichTextNode =
+  | Block
+  | Inline
+  | (Text & {
+      content?: RichTextNode;
+    });
+
+const extractNodeContent = (node: RichTextNode): ReactNode[] => {
+  const getContent = (): ReactNode[] =>
+    Array.isArray(node.content)
+      ? node.content.map(extractNodeContent).flat()
+      : [];
+
+  const isBold = (node: Text): boolean =>
+    node.marks.some(({type}: Mark) => type === MARKS.BOLD);
+
+  switch (node.nodeType) {
+    case 'text':
+      return node.value
+        ? [isBold(node) ? <b key={node.value}>{node.value}</b> : node.value]
+        : [];
+    case INLINES.HYPERLINK: {
+      const linkContent = getContent();
+      return [
+        <Link key={linkContent.join('-') + node.data.uri} href={node.data.uri}>
+          {linkContent}
+        </Link>,
+      ];
+    }
+    default:
+      return getContent();
+  }
+};
+
+const richTextRenderOptions: Options = {
+  renderNode: {
+    [BLOCKS.PARAGRAPH]: (paragraphNode: RichTextNode) => {
+      const paragraphContent = extractNodeContent(paragraphNode);
+      return paragraphContent.length ? (
+        <BodyTwoText>{paragraphContent}</BodyTwoText>
+      ) : (
+        <br />
```

Comment:
Curious why we are adding a break when there is no paragraph content? Wouldn't an empty `BodyTwoText` work?